src/main/java/org/squiddev/cobalt/Buffer.java

```diff
@@ -156,6 +156,20 @@ public final Buffer append(byte b) {
 		return this;
 	}
 
+	/**
+	 * Append an array of bytes to the buffer.
+	 *
+	 * @param b The bytes to append
+	 * @return {@code this} to allow call chaining
+	 */
+	public final Buffer append(byte[] b) {
+		makeRoom(0, b.length);
+		System.arraycopy(b, 0, bytes, offset, b.length);
+		offset += b.length;
```

Comment:
I don't think offset should grow, should it?

Reply:
Sorry, I thought offset was a pointer to the "write head". Fixed